build.sc

```diff
@@ -18,9 +20,16 @@ val scalaNativeVersions = for {
   scalaNativeV <- Seq("0.4.0")
 } yield (scalaV, scalaNativeV)
 
-trait UtestModule extends PublishModule {
+trait UtestModule extends PublishModule with Mima {
   def artifactName = "utest"
 
+  def mimaPreviousArtifacts = Agg.from(
+    VcsVersion
+      .vcsState()
+      .lastTag
+      .map(lastTag => ivy"com.lihaoyi::utest::$lastTag")
+  )
```

Comment:
This makes `mimaReportBinaryIssues` check for binary compatibility with the last tag on Git.